src/sql/src/plan/error.rs

```diff
@@ -270,6 +272,12 @@ impl PlanError {
             Self::DuplicateSubsourceReference { .. } => {
                 Some("Specify target table names using FOR TABLES (foo AS bar), or limit the upstream tables using FOR SCHEMAS (foo)".into())
             }
+            Self::InvalidKeysInSubscribeEnvelopeUpsert => {
+                Some("All keys must be columns on the underlying relation.".into())
+            }
+            Self::InvalidOrderByInSubscribeWithinTimestampOrderBy => {
+                Some("All order bys must be variables that are getting returned.".into())
```

Comment:
This message is a bit confusing. I do not think there are any "variables" in this context.

I guess what is Only column names allowed in the `ORDER BY` clause:

```
materialize=> COPY (SUBSCRIBE t1 WITH (SNAPSHOT=TRUE) WITHIN TIMESTAMP ORDER BY f1+f2) TO STDOUT;
ERROR:  invalid ORDER BY in SUBSCRIBE WITHIN TIMESTAMP ORDER BY
HINT:  All order bys must be variables that are getting returned.
```

Reply:
Yeah. changed to "All order bys must be output columns"